extensions/amp-access-laterpay/0.1/test/validator-amp-access-laterpay.html

```diff
@@ -0,0 +1,71 @@
+<!--
```

Comment:
This looks like an exact copy of the one for `amp-access`. Could this be updated with actual code that would be used in a amp-access-laterpay situation? Perhaps from the [markup](https://github.com/laterpay/amphtml/blob/eaf61346111e620c0ecd50326dda23f47b58f024/extensions/amp-access-laterpay/amp-access-laterpay.md) in #5920?